### PR TITLE
Add sql_writer.go and test

### PR DESCRIPTION
**Pull request by wangkuiyi**
This is part of the fix of https://github.com/sql-machine-learning/sqlflow/issues/1352

### CODE REVIEW

File: pkg/sqlfs/sql_writer_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	a.NoError(e1)
+	a.True(has)
+
+	a.NoError(dropTable(testDB, fn))
+}
+
+func TestSQLWriterWriteAndRead(t *testing.T) {
+	testDriver = getEnv("SQLFLOW_TEST_DB", "mysql")
+	if testDriver == "hive" {
+		return // For Hive, please use HiveWriter instead.
```
Comment (Yancey0623) [inline pkg/sqlfs/sql_writer_test.go:51]: Can use `t.Skip()` instead of `return` so that other developers can know to skip this test case clearly.

**Review by Yancey0623 — APPROVED** [review]
LGTM++

File: pkg/sqlfs/sql_writer.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		}
+		return nil
+	}
+}
+
+func noopWrapUp() error {
+	return nil
+}
+
+func newSQLWriter(db *sql.DB, driver, table string) (io.WriteCloser, error) {
```
Comment (Yancey0623) [inline pkg/sqlfs/sql_writer.go:48]: Do you want to rewrite the `Create` function in `writer.go` using `SQLWriter` and `HiveWriter`? If so, may be we can move `dropTable` and `createTable` in the `Create` function.

``` golang
func Create(db *sql.DB, driver, table string, session *pb.Session) (io.WriteCloser, error) {
    if e := dropTable(db, table); e != nil {...}
    if e := createTable(db, driver, table);  e != nil {...}
    if driver == "hive" {
        return newHiveWriter(...)
    }
    return newSQLWriter(...)
}
```

Comment (wangkuiyi) [reply pkg/sqlfs/sql_writer.go:48]: I agree. I think we will need to rewrite Create, maybe after we merge HiveWriter.

File: pkg/sqlfs/sql_writer_test.go (hunk 1 above)
Comment (wangkuiyi) [reply pkg/sqlfs/sql_writer_test.go:51]: Thanks for the remind! Will update in successive PRs.

**Pull request merged**